core/src/main/scala/anorm/ToStatement.scala

```diff
@@ -613,3 +661,25 @@ sealed trait JodaToStatement { // TODO: Move in a separate file
       else s.setNull(index, meta.jdbcType)
   }
 }
+
+sealed trait ToStatementPriority1 extends ToStatementPriority0 {
```

Comment:
It would be good to document here why we need different priority implicits, it's really hard to tell just by reading the code.  I'm guessing this is to ensure that `Array[Byte]` takes priority over `Array[T]`.